File: 프로그래머스/lv1/12930. 이상한 문자 만들기/이상한 문자 만들기.cs
using System;

public class Solution
    {
        public string solution(string s)
        {
            string answer = "";
            
            string[] temp = s.Split();
            
            for (int i = 0; i < temp.Length; i++)
            {
           
                for (int j = 0; j < temp[i].Length; j++)
                {
               
                    if (j == 0)
                    {
                        answer += Char.ToUpper(temp[i][j]);
                        continue;
                    }
                 
                    if (j % 2 == 0)
                    {
                        answer += Char.ToUpper(temp[i][j]);
                    }
                    else
                    {
                        answer += Char.ToLower(temp[i][j]);
                    }
                }
                
                if (i != temp.Length-1)
                {
                    answer += " ";
                }
            }
            
            return answer;
        }
    }

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    string answer = "";
    int index = 0;
    
    for(int i = 0; i < s.length(); i ++)
    {
        if(s[i] == ' ')
        {
            index = 0;
            continue;
        }
        
        if(index % 2 == 0)
        {
            if(s[i] >= 97 && s[i] <= 122)
            { s[i] -= 32;}
            
            
        }
        else
        {
            if(s[i] >= 65 && s[i] <= 96)
            {
                s[i] += 32;
                
            }
            //홀수면 소문자
        }
        
        index += 1;
        
    }
    answer = s;
    return answer;
}